docs/en/observability/rate-aggregations.asciidoc

```diff
@@ -0,0 +1,49 @@
+[[rate-aggregations]]
+= Rate aggregations
```

Comment:
I think aggregations should be singular (In [elasticsearch document rate aggregation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-rate-aggregation.html), it is singular)